server/models/favoriterecipe.js

```diff
@@ -0,0 +1,19 @@
+module.exports =  (sequelize, DataTypes) => {
+  const FavoriteRecipe = sequelize.define('FavoriteRecipe', {
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
+    recipeId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    }
+  }, {
+    classMethods: {
+      associate() {
+        // associations can be defined here
```

Comment:
I would love to see the associations defined here

Reply:
Ok. writing the association